codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ValueEntitySourceGenerator.scala

```diff
@@ -48,8 +50,17 @@ object ValueEntitySourceGenerator {
          |
          |$imports
          |
-         |class ${entity.fqn.name} /* extends Abstract${entity.fqn.name} */ {
+         |/** A value entity. */
+         |class ${entity.fqn.name}(val entityId: String) /* extends Abstract${entity.fqn.name} */ {
          |  ${Format.indent(methods, 2)}
+         |
+         |  // TODO override
+         |  def emptyState =
+         |    throw new UnsupportedOperationException("Not implemented yet, replace with your empty entity state")
+         |}
+         |object ${entity.fqn.name} {
+         |  def apply(context: Unit /* ValueEntityContext */):${entity.fqn.name} =
+         |   new ${entity.fqn.name}("TODO")
```

Comment:
This commit is unrelated.
I guess you were working on another PR that will bring that change already, correct?

Reply:
and probably the reason for ci failure